canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -11,68 +11,115 @@
     <header>
       <nav>
         <h3>Admin Console</h3>
+          <!-- UI comment - adding padding between header and the cards -->
```

Comment:
I think it would be cleaner to have space added by adding some padding to the `h3` instead of using a `&nbsp;` outside of a typical text-wrapping container like a span or div.